tests/tests_lvm_pool_shared.yml

```diff
@@ -0,0 +1,159 @@
+---
+- name: Test LVM shared pools
+  hosts: all
+  become: true
+  vars:
+    storage_safe_mode: false
+    storage_use_partitions: true
+    mount_location1: '/opt/test1'
+    volume1_size: '4g'
+
+  tasks:
+    # Skip this test if run on localhost.
+    # Local run will not set up lvmlockd
+    # correctly and the test would fail.
+    - name: Skip test if the control node of the managed node is localhost
+      meta: end_host
+      when: inventory_hostname == "localhost"
+
+    - name: Gather package facts
```

Comment:
```suggestion
    - name: Run the role to install blivet
      include_role:
        name: linux-system-roles.storage
      vars:
        storage_pools: []
        storage_volumes: []

    - name: Gather package facts
```

Reply:
otherwise, the task `Set blivet package name` as blivet is not installed.
Are you able to run this test locally on your laptop?

Reply:
I moved the initial storage role (which installs blivet) run before the check.
While running the test I also noticed that in HA cluster role test_setup.yml there is this task:

```
  - name: Set node name to 'localhost' for single-node clusters
    set_fact:
      inventory_hostname: localhost  # noqa: var-naming
    when: ansible_play_hosts_all | length == 1

```

I am not sure what its purpose is in the role but it messed up the test when I tried to run it on single remote node. I replaced it with task that changes inventory name from 'localhost' to '127.0.0.1' and it seems to do the trick.